include/deal.II/lac/parpack_solver.h

```diff
@@ -307,13 +313,19 @@ class PArpackSolver : public Subscriptor
    * Set shift @p sigma for shift-and-invert spectral transformation.
    *
    * If this function is not called, the shift is assumed to be zero.
+   *
+   * @note This function is only relevant for <code>mode=3</code>.
```

Comment:
-> only relevant for <code>mode=3</code> (see the general documentation of this class for a definition of what the different modes are).